blocks/full-bleed/full-bleed.css

```diff
@@ -0,0 +1,331 @@
+.full-bleed.block .container {
+  margin: 0 auto;
+}
+
+.full-bleed.block .credit {
+  margin: 0 auto;
+  padding: 15px;
+}
+
+.full-bleed.block .credit span {
+  color: #adb4b7;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 16px;
+  letter-spacing: 0;
+}
+
+.full-bleed.block .lead {
+  display: flex;
+  flex-direction: column-reverse;
+  position: relative;
+}
+
+@media (min-width: 769px) {
+  .full-bleed.block .lead {
+    display: grid;
+    grid-template-rows: 750px auto;
+    grid-template-columns:
+      1fr min(230px, 10%) min(400px, 40%) min(400px, 40%) min(230px, 10%)
+      1fr;
+  }
+}
+
+.full-bleed.block .first-paragraph {
+  color: blue;
+}
+
+.full-bleed.block .headline {
+  padding: 15px;
+}
+
+@media (min-width: 769px) {
+  .full-bleed.block .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    grid-column: 2 / span 4;
+    grid-row: 1;
+    height: 100%;
+    justify-content: flex-end;
+    z-index: 2;
+    text-align: center;
+    padding: 50px 110px 100px;
+  }
+}
+
+.full-bleed.block .headline .rubric {
+  font-size: 10px;
+  margin-bottom: 20px;
+  display: block;
+  color: #adb4b7;
+  font-weight: 500;
+  letter-spacing: 0.3em;
+  text-transform: uppercase;
+}
+
+@media (min-width: 769px) {
+  .full-bleed.block .headline .rubric {
+    font-size: 10px;
+  }
+}
+
+.full-bleed.block .headline > .title > h1 {
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: 0.05em;
+  text-align: center;
+}
+
+@media (min-width: 769px) {
+  .full-bleed.block .headline h1 {
+    text-align: center;
+    margin-top: 0;
+    margin-bottom: 20px;
+    font-size: 50px;
+    color: #fff;
+  }
+}
+
+.full-bleed.block .article-body div > p {
+  margin: 0 auto 30px;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 30px;
+}
+
+@media (min-width: 769px) {
+  .full-bleed.block .article-body div > p {
+    margin-bottom: 60px;
+  }
+}
+
+.full-bleed.block .attribution {
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: -0.03em;
+  font-size: 14px;
```

Comment:
@Lauraa1003 it looks like there should be a bigger font-size for desktop can you please double check ?